java/arcs/sdk/testing/BaseTestHarness.kt

```diff
@@ -90,7 +89,11 @@ open class BaseTestHarness<P : Particle>(
                 DriverAndKeyConfigurator.configureKeyParsers()
 
                 scope.launch {
-                    val handleManager = EntityHandleManager(HandleManager(JvmTime))
+                    val handleManager = EntityHandleManager(
+                        "testHarness",
+                        "testHarnessHost",
```

Comment:
I'd suggest using named arguments whenever it isn't completely obvious what the arguments actually are (which is the case here, I feel)